app/templates/descriptor/index.html

```diff
@@ -7,43 +7,37 @@
 {% block content %}
     <div class="ui stackable grid container">
         <div class="sixteen wide tablet twelve wide computer centered column">
-            <h2 class="ui header">
+            <h2 class="centered ui header">
                 Descriptors
                 <div class="sub header">
                     View and edit current resource descriptors.
                 </div>
             </h2>
 
-            <div class="ui menu">
-                <div id="select-type" class="ui dropdown item">
-                    <div class="text">
-                        All descriptors
-                    </div>
-                    <i class="dropdown icon"></i>
-                    <div class="menu">
-                        <div class="item" data-value="">All Descriptors</div>
-                        <div class="item" data-value="Text">Text</div>
-                        <div class="item" data-value="Option">Options</div>
-                    </div>
-                </div>
-                <div class="ui right search item">
-                    <div class="ui transparent icon input">
-                        <input id="search-descriptors" type="text" placeholder="Search descriptors…">
-                        <i class="search icon"></i>
-                    </div>
-                </div>
+            <!--waiting for functionality-->
+            <div class="ui right action left icon input menu">
+                <i class="search icon"></i>
+                    <input id="search-descriptors" type="text" placeholder="Search descriptors">
+                    <!--<div class="ui basic floating dropdown button">
+                        <div class="text">All Descriptors</div>
+                        <i class="dropdown icon"></i>
+                        <div class="menu">
+                            <div class="item" data-value="">All Descriptors</div>
+                            <div class="item" data-value="Text">Type</div>
+                            <div class="item" data-value="Option">Values</div>
+                        </div>
+                    </div>-->
             </div>
 
             {# Use overflow-x: scroll so that mobile views don't freak out
              # when the table is too wide #}
             <div style="overflow-x: scroll;">
-                <table class="ui searchable sortable unstackable selectable celled table">
```

Comment:
meant to add `searchable` back in but didn't realize that the fix was overwritten
added `striped` for style (to make it easier